x/interchainqueries/types/message_remove_interchain_query.go

```diff
@@ -7,24 +7,22 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-var _ sdk.Msg = &MsgRemoveInterchainQueryRequest{}
```

Comment:
Why is this removed?
It's pretty necessary compile check that `MsgRemoveInterchainQueryRequest` really implements sdk.Msg

Reply:
that happened accidentally, tnx